modules/rosa-sdpolicy-am-billing.adoc

```diff
@@ -4,8 +4,11 @@
 // * rosa_architecture/rosa_policy_service_definition/rosa-service-definition.adoc
 :_mod-docs-content-type: CONCEPT
 [id="rosa-sdpolicy-billing_{context}"]
-= Billing
+= Billing and pricing
 
-{product-title} is billed through Amazon Web Services (AWS) based on the usage of AWS components used by the service, such as load balancers, storage, EC2 instances, other components, and Red{nbsp}Hat subscriptions for the OpenShift service.
+ROSA is billed directly to your AWS account. ROSA pricing can be consumption based, with annual commitments or three-year commitments for greater discounting. The total cost of ROSA consists of two components:
```

Comment:
Just a small wording suggestion
```suggestion
{product-rosa} is billed directly to your {aws-first} account. ROSA pricing is based on an annual commitment or three-year commitment. The total cost of ROSA consists of two components:
```

Reply:
If you choose to keep it the way it is I would change "can be" to "is" is it is based on the commitment you choose.
```suggestion
product-rosa} is billed directly to your {aws-first} account. ROSA pricing is consumption based, with annual commitments or three-year commitments for greater discounting. The total cost of ROSA consists of two components:
```